src/common/config.hpp

```diff
@@ -45,23 +50,16 @@ struct SensorMessage {
   bool towerside_armed;
   bool has_contact;
   // Ignition currents
-  uint16_t ignition_primary_ma;
-  uint16_t ignition_secondary_ma;
+  uint16_t ignition_ma;
   // Actuator states
-  ActuatorPosition::ActuatorPosition ov101_state;
-  ActuatorPosition::ActuatorPosition ov102_state;
-  ActuatorPosition::ActuatorPosition ov103_state;
-  // Tank Heating
-  uint16_t heater_thermistor_1;
-  uint16_t heater_thermistor_2;
-  uint16_t heater_current_ma_1;
-  uint16_t heater_current_ma_2;
-  uint16_t heater_batt_mv_1;
-  uint16_t heater_batt_mv_2;
-  uint16_t heater_kelvin_low_mv_1;
-  uint16_t heater_kelvin_low_mv_2;
-  uint16_t heater_kelvin_high_mv_1;
-  uint16_t heater_kelvin_high_mv_2;
+  ActuatorPosition::ActuatorPosition NV102_state;
+  ActuatorPosition::ActuatorPosition NV103_state;
+  ActuatorPosition::ActuatorPosition NV104_state;
+  ActuatorPosition::ActuatorPosition NV105_state;
+  ActuatorPosition::ActuatorPosition OV102_state;
+  ActuatorPosition::ActuatorPosition FV102_state;
+  ActuatorPosition::ActuatorPosition OV101v_state;
+  ActuatorPosition::ActuatorPosition FV101v_state;
```

Comment:
There is no position feedback on the actuators

Reply:
But we do want to send the auto sequencing data to clientside like I talked about in slack.